src/html-duration-picker.js

```diff
@@ -187,7 +187,7 @@ export default (function() {
       const pickerLeftMargin = currentPickerStyle.marginLeft;
       const totalPickerWidth = currentPickerStyle.width;
       picker.setAttribute('data-upgraded', true);
-      picker.value = '00:00:00';
+      picker.value = picker.dataset.duration || '00:00:00';
```

Comment:
@PeterMorganGH you are on fire today 🔥 👏 👏 👏 thank you again.

Just a concern. How is the `Element.dataset` coverage across different browsers? Would it better if we used ```Element.getAttribute(.. ```? I'm just thinking out loud, I don't really know. 
I could not find information on the browser coverage. Would you happen to know?
According to cainiuse.com, coverage is 97.34%, but there are cases where I've found cainiuse to be unreliable

Reply:
Also please consider line 127. Could we have it fall back to the `data-duration` value too?

Reply:
As you mentioned, CanIUse shows high compatibility - even IE11. According to the [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Element/getAttribute#Notes) article on `getAttribute()` some older browsers will return an empty string instead of `null`. If using `getAttribute()`, the suggestion is to check for a value with `hasAttribute()` beforehand. That is a lot more code for the same result.

Reply:
I added the `dataset.duration` value to line 128.

Reply:
Excellent points, and very good work. Thank you once again @PeterMorganGH